examples/ct/ctmapper/lookup/lookup.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The lookup binary looks up a specific ID in a map.
```

Comment:
Does main still need the "Package x ..." format? There are other cases if so, if not you can ignore this.